src/markdown-pages/build-apps/create-an-ab-test-application/catalog-info.mdx

```diff
@@ -106,6 +106,8 @@ Save the [this screenshot](https://github.com/newrelic-experimental/nru-programm
 
 Users will be able to see a carousel of screenshots in your app's **About** page in the catalog.
 
+![Screenshot Carousel](../../../images/ab-test/ab-testing-carousel-preview.png)
```

Comment:
Could you elaborate on the two screenshot additions in this guide? The original intention here was not to have images because the user doesn't submit the catalog information until the [next guide](https://developer.newrelic.com/build-apps/ab-test/publish/#submit-catalog-information). I think this is a similar situation to the one @tyreer points out below.

Reply:
Makes sense, I can make those changes.